### PR TITLE
Support get topic applied policy for DeduplicationStatus

### PR DESCRIPTION
**Pull request by 315157973**
Master Issue: #9216

### Modifications
1) The api name of topic-level is consistent with that of namespace-level
2) Fix the problem that the namespace-level policy cannot be removed
3) Fix the problem that topic-level does not take effect when multiple levels of policy are set at the same time
4) Added applied API

### Verifying this change
Verify that the new API is correct
Verify that the applied API is correct
Verify that policies of different levels exist at the same time and whether the priority is correct

### CODE REVIEW

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/Namespaces.java [hunk 1]
```diff
@@ -335,6 +335,16 @@ public void setSubscriptionExpirationTime(@PathParam("tenant") String tenant,
         internalSetSubscriptionExpirationTime(expirationTime);
     }
 
+    @GET
+    @Path("/{tenant}/{namespace}/deduplication")
```
Comment (zymap) [inline pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/Namespaces.java:339]: We already have policy rest API. The `deduplication` is a property in the policy. Why do we need to introduce a new Rest API for this?

Comment (315157973) [reply pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/Namespaces.java:339]: We do not yet have an interface to return the entire policies, but I don't think it is a good way to return the entire policies every time.

Comment (zymap) [reply pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/Namespaces.java:339]: We have the get namespace policy API: https://github.com/apache/pulsar/blob/7c6f5e2bbc8b753609b89a7589f480a57c711260/pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/Namespaces.java#L119

Comment (315157973) [reply pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/Namespaces.java:339]: The user can get entire policies, or only a single piece of policies, there is no conflict

Comment (315157973) [reply pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/Namespaces.java:339]: The documentation we give to users does not say that every interface returns all the data. Although we can handle it in the SDK, the user may directly call the API.
So I think it is better to open the interface separately.

**Comment by sijie** [comment]
@zymap Can you review it again?

**Review by codelipenghui — APPROVED** [review]
Looks good to me, could you please rebase the master branch?

**Review by zymap — APPROVED** [review]
LGTM

**Pull request merged**